src/styles/blocks/interesting-articles.css

```
@@ -41,6 +41,33 @@
   color: red;
 }
 
+.interesting-articles__add-article {
+  padding: 12px;
+  font-weight: 500;
+  border: 1px solid black;
+  background-color: #fcffe7;
+  margin: 40px 0;
+  width: 33%;
+}
+
+.interesting-articles__article-preview,
+.interesting-articles__form {
+  text-align: left;
+}
+
+.interesting-articles__form {
+  width: 66%;
+  margin: 0 auto;
+}
```

тут лучше делать таким образом
например 834px это ширина в форме
```suggestion
.interesting-articles__form {
  max-width: 834px;
  width:100%;
  margin: 0 auto;
}
```

или через гриды
в фигме можно сетку включить
https://i.gyazo.com/af1a7a1c6cf038b5615275cec88d430d.png
layots grid
и соо-о можно задать гриды, например grid-column-start 3 и grid-column-end: 11